### PR TITLE
OpenThread RCP mode integration into Zephyr

### PR DESCRIPTION
**Pull request by markus-becker-tridonic-com**
* Change sample prj.conf for RCP

Relates to https://github.com/zephyrproject-rtos/zephyr/issues/26383.

Depends on https://github.com/zephyrproject-rtos/openthread/pull/30 and https://github.com/nrfconnect/sdk-zephyr/pull/356.

### CODE REVIEW

**Comment by MarekPorwisz** [comment]
@rlubos FYI

**Review by MarekPorwisz — CHANGES_REQUESTED** [review]
Hi Markus,

First of all, thank you for your contribution.
We would like to keep ncp sample functioning in parallel to rcp. What I would suggest doing here is creating separate overlay file  that converts ncp sample into rcp. You can see an example of how to use configuration overlay files in our coap client sample (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/openthread/coap_client/README.html https://github.com/nrfconnect/sdk-nrf/tree/master/samples/openthread/coap_client ).

File: samples/openthread/ncp/prj.conf [hunk 1]
```diff
@@ -17,12 +20,20 @@ CONFIG_OPENTHREAD_LOG_LEVEL_CRIT=y
 CONFIG_NET_CONFIG_LOG_LEVEL_OFF=y
 CONFIG_LOG_MINIMAL=n
 CONFIG_LOG_BACKEND_SPINEL=y
-CONFIG_LOG_BACKEND_RTT=n
+CONFIG_LOG_BACKEND_RTT=y
```
Comment (MarekPorwisz) [inline samples/openthread/ncp/prj.conf:23]: I don't think this should be enabled by default.

**Comment by markus-becker-tridonic-com** [comment]
This was only created to share an example config with you. You would need to create a config similar to this, when you downstream the zephyr changes. Closing, since https://github.com/nrfconnect/sdk-zephyr/pull/356 is also closed now.

**Pull request closed without merging**